docs/design/2023-09-22-tiflash-query-level-auto-spill.md

```diff
@@ -0,0 +1,55 @@
+# TiFlash query level auto spill
+
+- Author: [windtalker](https://github.com/windtalker)
+
+## Introduction
+
+This document introduce the design and implemantion of TiFlash's query level auto spill framework.
+
+## Motivation and Background
+
+TiFlash already support operator level auto spill from V7.0.0, some memory intensive opeartors can auto spill if the memory usage exceeds threshold. However, there are some shortcomings for the operator level auto spill
```

Comment:
```suggestion
TiFlash have already supported operator level auto spill since V7.0.0, some memory intensive opeartors can auto spill if the memory usage exceeds threshold. However, there are some shortcomings for the operator level auto spill
```